docs/technical-reference.md

```diff
@@ -266,15 +266,15 @@ to Google's OAuth 2.0 server.
 ![](./imgs/google_console.png)
 
 Copy the `Client ID` and `Client secret` to a safe place.
-Add `YOUR_QUILT_CATALOG_URL/oauth-callback` to *authorized redirect URIs*.
+Add `QuiltWebHost/oauth-callback` to *authorized redirect URIs*.
```

Comment:
This changeset is helpful. How can we distinguish literal text from a template? Maybe `$QuiltWebHost` or `< QuiltWebHost>`? Go with the one that you think is most consistent and clear before merging. Pre-emptively approving.

Reply:
This changeset was pushed from the wrong branch. It has been reverted.